src/components/docHistory/DocHistory.tsx

```diff
@@ -5,100 +5,30 @@ import styled from 'styled-components';
 import { VscTriangleDown } from 'react-icons/vsc';
 import { getDocHistories } from '@/apis/history';
 import { useSearchParams } from 'next/navigation';
+import { IHistoryData } from '@/types/request';
+import { parseAndFormatDate, renderFirstStr, renderNewStr, renderOldStr } from '@/utils/historyUtils';
+import { useHistoryQuery } from '@/hooks/useHistoryQuery';
 
-interface HistoryData {
-	title: string;
-	author: string;
-	created_at: string;
-	content: string;
-	change: string;
-}
 
 const DocHistory = () => {
 	const params = useSearchParams();
-	const title = params.get('title');
-	const [dataList, setDataList] = useState<HistoryData[]>();
+	const [title, setTitle] = useState('');
+	const dataList = useHistoryQuery(title).data;
```

Comment:
isLoading을 따로 안쓰면 useHistoryQuery 훅에서 data만 반환해도 될 것 같아요

Reply:
만약 isLoading을 쓴다고 하면 분해할당으로 받으면 깔끔해집니다
```suggestion
	const { data, isLoading } = useHistoryQuery(title);
```

Reply:
이건 사소하지만 데이터를 유추할 수 있는 변수명이 좋은 것 같아요
```suggestion
	const {data: docHistoryList, isLoading }= useHistoryQuery(title);
```

Reply:
오 너무 좋아요 코드 참고하면서 제 코드에서는 isLoading이 불필요하다고 느꼈는데 그냥 빼면 되는군요~
삭제하고 변수명도 수정했습니다!